libbpf-cargo/src/test.rs

```diff
@@ -785,7 +785,8 @@ fn test_skeleton_builder_clang_opts() {
         .debug(true)
         .clang("clang")
         .build_and_generate(skel.path())
-        .unwrap_err();
```

Comment:
`unwrap_err` requires `Debug` on the Ok side of the result. I didn't want to implement that just for this test on `CompilationOutput`, `.err().unwrap()` achieves the same thing without needing to print the Ok type.